README.md

```diff
@@ -48,34 +47,26 @@ To develop an ERDOS application in Python, simply run
 
 If you'd like to contribute to ERDOS, first
 [install Rust](https://www.rust-lang.org/tools/install).
-Then run the following to clone the repository and build ERDOS:
+Within a [virtual environment](https://docs.python.org/3/tutorial/venv.html),
+run the following to clone the repository and build ERDOS:
 ```console
-rustup default nightly  # use nightly Rust toolchain
-git clone https://github.com/erdos-project/erdos.git && cd erdos
-python3 python/setup.py develop
+git clone https://github.com/erdos-project/erdos.git && cd erdos/python
+pip3 install maturin
```

Comment:
We should maintain a `requirements.txt`.

Reply:
I disagree -- the `pyproject.toml` already specifies the package dependencies. With newer pip versions, you can even run `pip install -e .` from the `python/` dir and it'll install without having installed `maturin` ahead of time.

Reply:
Did not know that! Okay fair enough, I'll resolve this and the corresponding conversation then. Thanks!